docs/SUMMARY.md

```diff
@@ -0,0 +1,63 @@
+# Summary
+
+## Getting started
+
+* [Deployment guide](DEPLOYMENT.md)
+* [Usage guide](USAGE.md)
+
+## JSON-RPC clients
+
+* [Golang client](https://godoc.org/github.com/0xProject/0x-mesh/rpc)
+* [Typescript client](json_rpc_clients/typescript/README.md)
```

Comment:
nit: can we call it "TypeScript" with a capital "S" to match the spelling on the [official website](https://www.typescriptlang.org/)?